NEXT_CHANGES.rst

```diff
@@ -11,3 +11,54 @@ links to relevant Issues, Discussions, and PR's on github with the following for
 `Issue #XX <https://github.com/ixdat/ixdat/issues/XX>`_
 
 `PR #XX <https://github.com/ixdat/ixdat/pulls/XX>`_
+
+
+Debugging for 0.2.1
+-------------------
+
+Will add the following when we finish debugging (will be 0.2.1 then). But some will go
+in the 0.2.0 section of CHANGES.rst as indicated:
+
+API changes
+...........
+
+techniques
+^^^^^^^^^^
+- Lookups instead of properties (as of 0.2.0)
+
+  - ``my_ec_meas["raw_potential"]`` replaces ``ECMeasurement.raw_potential``
+  - ``my_ec_meas["raw_currents"]`` replaces ``ECMeasurement.raw_potential``
+  - ``my_cv["scan_rate"]`` replaces ``CyclicVoltammogram.scan_rate``
+
+  The old properties are deprecated.
+
+- ``ECMSCyclicVoltammogram.diff_with()`` raises a ``NotImplementedError`` rather than
+  an obscure error message. (as of 0.2.1)
+
+- ``ECMSPlotter.plot_vs_potential`` can accept ``color`` as a keyword argument. This
+  colors both the U-J curve in the lower panel and all the signals in the top panel,
+  so best to use with together with a ``mass_list`` or ``mol_list`` of length 1. (as of 0.2.1)
+
+exporters
+^^^^^^^^^
+
+- ``export()`` functions now take a ``time_step`` argument. It must be used together with
+  a ``tspan``. It is effective in reducing the file size of exported .csv's.
+
+- Renamed keyward argument: ``columns`` replaces ``v_list in exporters (``ECExporter`` in v0.2.0, ``ECMSExporter`` in 0.2.1)
+
+  The old name is deprecated.
+
+- ``ECMSExporter.export()`` functions now take a ``time_step`` argument. It must be used together with
+  a ``tspan``. It is effective in reducing the file size of exported .csv's.
```

Comment:
Is this a duplicate of line 45&46 (which seems to be more general)

Reply:
yes. fixed.